e3nn/__init__.py

```diff
@@ -24,6 +26,7 @@ def set_optimization_defaults(
     """
     _OPT_DEFAULTS['specialized_code'] = specialized_code
     _OPT_DEFAULTS['optimize_einsums'] = optimize_einsums
+    _OPT_DEFAULTS['jit_script_fx'] = jit_script_fx
```

Comment:
My fault, but this method makes no sense... it should only be updating keys that are actually passed as parameters, and shouldn't touch others.

Something with `**kwargs`, check that keys are a subset of those in _OPT_DEFAULTS, and then `_OPT_DEFAULTS.update(kwargs)`...?

Reply:
done